content/mtr/mtr.md

```diff
@@ -20,232 +20,228 @@ In addition to rules defined in [MTR - 5.4 - Unsporting Conduct](https://blogs.m
 
-- Player A has a win attempt, player B has a win attempt that player A knows about and player C only has one answer to stop A or B but not Both. In this situation, player A cannot just try to win and use the argument that if player C uses their answer, it would be king making.
+### Tie Breakers
 
-Keep in mind that sometimes a player might seem to be performing a “king making” or “spite play” decision, when in reality that line of play is the one that favours them most in the tournament progression, for example:
+All player’s tie breakers will be calculated the same regardless if a player received a bye or not. Any player receiving a bye will have 3 opponents added to their opponent history with a .2 win rate percentage. The lower limit for a tie breaker is updated to .2 for all players
 
-- During Swiss rounds, player A presents a win, player B has a Pact of Negation with no mana to pay for the trigger on their next upkeep. However player B believes that by stopping player A they might have a chance that the game ends in a draw therefore resulting in them gaining 1 point instead of 0.
-- Player A presents a win, and they have Force of Will that player B saw in a previous turn via Gitaxian Probe, player B has a Pact of Negation, with no way to pay for the trigger in the next upkeep, but player B suspects that player C has interaction. Because player B is the first to respond, they must play the Pact of Negation so that player A uses their Force of Will and then allows player C to effectively counter the win.
+### Swiss Rounds
 
-### Intentionally Drawing Games
+This will be subject to player attendance, but the Tournament Organiser reserves the right to adjust the following to meet venue/time constraints. Adjustments may not be available when hosting qualifiers that adopt rules from other TOs.
 
-While it is completely acceptable that players decide to Intentionally Draw a game, and in some situations, it can be an elegant solution to an apparently “King-making” situation, there are some implications, when it comes to tournament logistics and player conduct:
+|Players|Minimum Number of Swiss Rounds|Playoff|
+|-|-|-|
+|1-4|None (run 1 single elimination round)|None|
+|5-15|2|Top 4|
+|16-24|3|Top 10|
+|25-32|4|Top 10|
+|33-64|5|Top 16|
+|65-128|6|Top 16|
+|129-256|7|Top 40|
+
+Each swiss round lasts 80 minutes (active player finishes their turn, maximum 20 minutes) or 75 minutes (one extra turn per active player). This decision is down to the TO but should take into consideration venue opening times etc.
```

Comment:
This is where we might want to put in some pros/cons for each choice to help TOs make their decision for which they want to use. I think most of our established events use 80 minutes but we should still give some reasoning for TOs coming from other formats so they can make an informed choice for themselves. This might also go in the [TO doc](https://cedh-uk.github.io/mtr-ipg/to/to/) that Ben has been working on